content/articles/Introduction-to-Containers-and-Docker/index.md

```diff
@@ -0,0 +1,87 @@
+
+> Because containerization is a Linux OS feature, docker can only be installed on Linux operating systems such as Ubuntu, Fedora, Redhat, and so on. If you wish to use docker on a Windows operating system, you'll need to create a Linux virtual machine. The Docker Windows Application creates a virtual computer and runs the Docker engine on top of it automatically.
+
+In a client-server architecture, Docker or Docker Engine is made up of three components:
+
+1. A daemon process on a server (docker)
+
+2. An API that allows applications to communicate with the Docker daemon process.
+
+3. A Docker command-line interface (CLI), that allows us to run docker commands. This allows us perform various tasks on the docker daemon process.
+ 
+The API is used by the command-line interface to interact with the Docker daemon process. Docker objects such as containers, images, networks, and volumes are created and managed by the docker daemon.
+
+### Reasons why we use dockers
+
+Recently, Docker has grown highly popular in the software industry. It lets developers and system admins construct and operate applications in containers, thanks to the growing need for microservices and DevOps. Here are some of the factors that contribute to Docker's popularity:
+
+1. **It's Versatile**: You may execute a basic hello world program within a docker container, a web server such as Apache HTTP Server, or Nginx within a docker container, any heavy-weight application within a docker container, and even an operating system within a docker container. Although Docker does not advise running an operating system within a container, it is still possible to do so.
+
+2. **Containers in Docker are freely linked.**: A container is a self-contained entity with its resource quota, networking configuration, and other features that make it easier for system administrators to replace or update one container without impacting the others.
+
+3. **Highly secure**: Docker's Highly Secure feature ensures that containers are entirely isolated from outside processes.
+
+4. **Docker containers are small and portable**: Docker containers are lighter than virtual machines since they just establish separate userspace and use the underlying OS Kernel of the system on which they are run.
+
+5. **It is portable**: When we finish building an application on our local workstation and need to deploy it to production, we run into several problems with environment configuration. However, if you use Docker, you can describe the container setup procedures, and Docker will ensure that the environment is set up the same way every time you run it.
+
+### Docker Windows framework necessities and establishments
+
```

Comment:
A title like "Docker Desktop for Windows requirements and installation" would be more appropriate